pkg/network/tracer/connection/kprobe/tracer.go

```diff
@@ -174,10 +167,6 @@ func (t *kprobeTracer) Start(closeFilter func(*network.ConnectionStats) bool) (e
 }
```

Comment:
Why we don't call t.Stop() if the manager can't start (line 171 or 164)
```
	if err := t.m.Start(); err != nil {
**t.Stop()**
		return fmt.Errorf("could not start ebpf manager: %s", err)
	}
```

Reply:
We probably should in order to shutdown the `tcpCloseConsumer` properly.

Reply:
@brycekahle oh, actually you're right, we should close the manager before the `tcpCloseConsumer`. I'll update `t.Stop()`